package.json

```diff
@@ -32,7 +32,7 @@
       "eslint --fix --max-warnings 0",
       "git add"
     ],
-    "!(**/*dist/**/*).{js,jsx}": [
+    "!(eslintrc.js) && !(**/*dist/**/*).{js,jsx}": [
```

Comment:
I think that this is getting unwieldy.  Can we please use a `.eslintignore` file?